include/range/v3/view/transform.hpp

```diff
@@ -517,8 +517,33 @@ namespace ranges
             }
         };
 
-        /// \section ex Example usage
+        /// # ranges::views::transform
+        /// The transform view takes in a function `T -> U` and converts an input
+        /// range of `T` into an output range of `U` by calling the function on every
+        /// element of the input range.
+        ///
+        /// ## Example
         /// \snippet example/view/transform.cpp transform example
+        ///
+        /// ## Syntax
+        /// ```cpp
+        /// auto output_range = input_range | ranges::views::transform(transform_func);
+        /// ```
+        ///
+        /// ## Parameters
+        /// <pre><b>transform_func</b></pre>
+        ///   - Maps an input value to an output value (`transform_func(T) -> U`)
+        ///
+        /// <pre><b>input_range</b></pre>
+        ///   - The range of elements to transform
```

Comment:
maybe add that Reference type is `T` like you do for `output_range`

Reply:
Done